python/lsst/faro/measurement/TractMeasurement.py

```diff
@@ -113,6 +113,14 @@ def run(
     ):
         data = defaultdict(list)
         for catalog, photoCalib, astromCalib, dataId in zip(catalogs, photoCalibs, astromCalibs, dataIds):
+            if self.config.requireAstrometry and astromCalib is None:
+                self.log.info("requireAtrometry is True but astromCalib is None for %s.  Skipping...",
```

Comment:
How common are these logs going to be?

Also, `requireAtrometry` is missing an s.

Reply:
I'm assuming pretty rare (or we've got bigger problems!)

Typo fixed...thanks for another good catch!